app/controllers/api/users/responsibilities_controller.rb

```diff
@@ -1,4 +1,4 @@
-class ResponsibilitiesController < Api::BaseController
+class ResponsibilitiesController < Api::Users::ResponsibilitiesController
```

Comment:
Oops sorry if I confused you, it should be: `Api::Users::ResponsibilitiesController < Api::BaseController`.
We're still extending the parent `Api::BaseController`, but we need to make sure we also scope our `ResponsibilitiesController` under `Api::Users`!